sdk/logical/events.go

```diff
@@ -5,10 +5,22 @@ package logical
 
 import (
 	"context"
+	"encoding/json"
+	"errors"
 
 	"github.com/hashicorp/go-uuid"
+	"google.golang.org/protobuf/types/known/structpb"
 )
 
+// common event metadata keys
+const (
+	EventMetadataSecretPath        = "secret_path"
+	EventMetadataOperation         = "operation"
+	EventMetadataSourcePluginMount = "source_plugin_mount"
```

Comment:
Good call. I realize that the `source_plugin_mount` one doesn't make much sense anymore (since that information is all in the `pluginInfo` bundle available with the event).